test/functionalTest/harnessFunctions.sh

```diff
@@ -94,32 +94,45 @@ function dMsg()
 #
 function dbInit()
 {
-  role=$1  
-  db=$1
+  role=$1
+  
+  host="${CB_DATABASE_HOST}"
```

Comment:
It seems to me that this is so that the MongoDB instance can be on a host different than localhost for the harness tests. Which is the case when running docker.